user.go

```diff
@@ -37,7 +37,7 @@ type User struct {
 	UnconfirmedEmail string           `jsonapi:"attr,unconfirmed-email"`
 	Username         string           `jsonapi:"attr,username"`
 	V2Only           bool             `jsonapi:"attr,v2-only"`
-	IsSiteAdmin      *bool            `jsonapi:"attr,is-site-admin"`
+	IsAdmin          *bool            `jsonapi:"attr,is-admin"`
```

Comment:
Can we keep both changes and document the deprecation? My thinking is that users of older versions of TFE will still need this.

```suggestion
        // Deprecated: IsSiteAdmin was deprecated in v202405-1 and will be removed in a future version of Terraform Enterprise
        IsSiteAdmin      *bool            `jsonapi:"attr,is-site-admin"`
	IsAdmin          *bool            `jsonapi:"attr,is-admin"`
```

Reply:
sure 
When we are ready to remove `is-site-admin` two PRs will be made: 1. to drop the attribute in `go-tfe` and then 2. to drop the attribute in `atlas`